app/routes/calendar/new.tsx

```diff
@@ -287,14 +284,36 @@ function DescriptionTextarea() {
   );
 }
 
+/**
+ * Retrieves a new Date object that is offset by several hours.
+ *
+ * NOTE: it is important that we work with & return a copy of the date here,
+ *  otherwise we will just be mutating the original date passed into this function.
+ */
+function getDateWithHoursOffset(date: Date) {
+  const DATE_INPUT_HOURS_OFFSET = 24;
+
+  const copiedDate = new Date(date.getTime());
+  copiedDate.setHours(date.getHours() + DATE_INPUT_HOURS_OFFSET);
+  return copiedDate;
+}
```

Comment:
This sounds like a great candidate to put into the `utils/date.ts` file where we are also importing the other utils from :) Maybe the offset could be a function parameter?

Reply:
That sounds like a good idea, I'll port it over there & add your suggestion. I only set the offset as a constant here because I did not anticipate it to be reused haha, but I will do the refactor